rest/access_test.go

```diff
@@ -949,17 +983,17 @@ func TestChannelAccessChanges(t *testing.T) {
 	dbc := rt.ServerContext().Database(ctx, "db")
 	database, _ := db.GetDatabase(dbc, nil)
 
-	collection := database.GetSingleDatabaseCollectionWithUser()
+	collectionUser := database.GetSingleDatabaseCollectionWithUser()
```

Comment:
A bit of a nit, but 'collectionUser' sounds like this is a user object, when it's actually a DatabaseCollectionWithUser.  Either 'collection' or 'collectionWithUser' would be better.